inc/config.class.php

```diff
@@ -1796,122 +1796,21 @@ function showPerformanceInformations() {
       echo "</table></div>";
    }
 
-   /**
-    * Display a HTML report about systeme information / configuration
-   **/
-   function showSystemInformations() {
-      global $DB, $CFG_GLPI;
-
-      if (!Config::canUpdate()) {
-         return false;
-      }
-
-      $rand = mt_rand();
-
-      echo "<div class='center' id='tabsbody'>";
-      echo "<form name='form' action=\"".Toolbox::getItemTypeFormURL(__CLASS__)."\" method='post' data-track-changes='true'>";
-      echo "<table class='tab_cadre_fixe'>";
-      echo "<tr><th colspan='4'>" . __('General setup') . "</th></tr>";
-
-      echo "<tr class='tab_bg_2'>";
-      echo "<td><label for='dropdown_event_loglevel$rand'>" . __('Log Level') . "</label></td><td>";
+   public static function showSystemInfoTable($params = []) {
+      global $CFG_GLPI, $DB;
 
-      $values = [
-         1 => __('1- Critical (login error only)'),
-         2 => __('2- Severe (not used)'),
-         3 => __('3- Important (successful logins)'),
-         4 => __('4- Notices (add, delete, tracking)'),
-         5 => __('5- Complete (all)'),
+      $p = [
+         'word_wrap_width' => 128
```

Comment:
Not sure that word wrapping is still usefull. Maybe it can be removed.

Reply:
It does make the display a bit nicer but it is completely dependent on the screen size since the table resizes.

Reply:
Maybe it could be removed and handled using a `whitespace: pre-wrap` style.

Reply:
It doesn't have the same effect. For example, the list of extensions in the PHP info overflows into multiple lines. With wordwrap, it breaks it at a specific width and indents the extra lines to make them stand out more.

The white-space property (any value) does not do this.